src/games/hangman/definitions.ts

```diff
@@ -7,6 +7,7 @@ export interface PlayerState {
   secretLength: number;
   hint?: string;
   guesses: Guesses;
+  declare?: string;
```

Comment:
Thank you for this PR! 

Why do you need to create this new field in the game state? Whats the difference between this and `secret`?